merge/merger.go

```diff
@@ -30,8 +31,77 @@ func NewMerger(fhirHost string) *Merger {
 // is successful a new FHIR Bundle containing the merged patient record is returned.
 // If a merge fails, a FHIR Bundle containing one or more OperationOutcomes is
 // returned detailing the merge conflicts.
-func (m *Merger) Merge(source1, source2 string) (mergeID string, outcome *models.Bundle, err error) {
-	return mockMerge(source1, source2)
+func (m *Merger) Merge(source1ID, source2ID string) (mergeID string, outcome *models.Bundle, err error) {
```

Comment:
The problem with that is I have my struct instances named those names already. I'll rename my objects and stick with what you provided :)